src/test/integration/tests/basic.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {html} from 'lit-html';
+import { SSRTest } from './ssr-test';
+
+export const tests = {
+  'textExpression': {
+    render(x: any) {
+      return html`<div>${x}</div>`;
+    },
+    expectations: [
+      {
+        args: ['TEST'],
+        html: '<div>TEST</div>'
+      },
+      {
+        args: ['TEST2'],
+        html: '<div>TEST2</div>'
+      }
+    ],
+    stableSelectors: ['div'],
+  },
+  'attributeExpression': {
+    render(x: any) {
+      return html`<div class=${x}></div>`;
+    },
+    expectations: [
+      {
+        args: ['TEST'],
+        html: '<div class="TEST"></div>'
+      },
+      // Attribute hydration not working yet
```

Comment:
Should we go ahead and add a `expectedToFail: true` or something similar, since we want to flush out a bunch of presumed-to-fail tests?

Reply:
I added skip and only options in #28